docs/user/_index.md

```diff
@@ -1,87 +1,89 @@
 +++
-title = "Grafana Agent documentation"
+title = "Grafana Agent"
 weight = 1
 +++
 
 # Grafana Agent
 
-Grafana Agent is a telemetry collector for sending metrics, logs,
-and trace data to the opinionated Grafana observability stack. It works best
-with:
+Grafana Agent collects and forwards telemetry data to open source deployments of the LGTM (Loki, Grafana, Tempo, Mimir) Stack, Grafana Cloud, or Grafana Enterprise, where your data can then be analyzed. You can install Grafana Agent on Kubernetes and Docker, or as a system process for Linux, macOS, and Windows machines.  
 
-* [Grafana Cloud](https://grafana.com/products/cloud/)
-* [Grafana Enterprise Stack](https://grafana.com/products/enterprise/)
-* OSS deployments of [Grafana Loki](https://grafana.com/oss/loki/), [Prometheus](https://prometheus.io/), [Cortex](https://cortexmetrics.io/), and [Grafana Tempo](https://grafana.com/oss/tempo/)
+The Grafana Agent is open source and its source code is available on GitHub at (https://github.com/grafana/agent).
```

Comment:
It looks like the parenthesis need to be removed here:

```suggestion
The Grafana Agent is open source and its source code is available on GitHub at https://github.com/grafana/agent.
```

Reply:
done